CHANGES.md

```diff
@@ -31,6 +31,8 @@
 * Now recognising new custom color maps from xcube server, for details 
   refer to https://github.com/xcube-dev/xcube/issues/1046. (#392)
 
+* Users can now copy snapshots of a map into the clipboard by clicking a new camera icon on a map's action bar.(#290)
+

```

Comment:
This is the section for the changes of version 1.3, which has already been release. Please align your branch with `main` first. Then I'll do another review.

Also limit lines to 80 characters in markdown.

Reply:
Done